tests/testsuite/git_commit.rs

```diff
@@ -0,0 +1,42 @@
+use ansi_term::Color;
+use std::process::Command;
+use std::{io, str};
+
+use crate::common::{self, TestCommand};
+
+#[test]
+fn test_render_commit_hash() -> io::Result<()> {
+    let repo_dir = common::create_fixture_repo()?;
+
+    let mut git_output = Command::new("git")
+        .args(&["rev-parse", "HEAD"])
+        .current_dir(repo_dir.as_path())
+        .output()?
+        .stdout;
+    // 7 bytes hexed encoded equals 14 chars
+    git_output.truncate(14);
```

Comment:
As mentioned earlier, I think a more reasonable default for the average user would be to have an output of 7 characters, rather than 7 bytes.

Reply:
updated to 7 chars.